src/lib/clients/system.ts

```diff
@@ -53,6 +55,37 @@ export function openUrl(url: string): Promise<ChildProcess> {
   return open(url);
 }
 
+export async function getVersion(toolName: string): Promise<string> {
+  try {
+    const toolResponse = await asyncExec(`${toolName} --version`);
+
+    if (toolResponse.stderr) {
+      throw new Error(toolResponse.stderr);
+    }
+
+    try {
+      const versionMatch = toolResponse.stdout.match(/(\d+\.\d+\.\d+)/);
+      if (versionMatch) {
+        return versionMatch[1];
+      }
+    } catch (err) {
+      throw new Error(`Could not parse ${toolName} version.`);
+    }
+  } catch (error) {
+    throw new Error(`Error getting ${toolName} version.`);
+  }
+
+  return "";
+}
+
+export async function checkVersion(minVersion: string, toolName: string): Promise<void> {
```

Comment:
@denishacquin this method makes sense to be in the simulator service, we are checking the simulator version requirements.

Reply:
OK